udata_front/theme/gouvfr/assets/js/components/AdminTable/AdminDatasetsTable/AdminDatasetsTable.vue

```diff
@@ -0,0 +1,216 @@
+        <AdminTableTh scope="col">
+          <Tooltip>
+            <span class="fr-icon--sm fr-icon-chat-3-line" aria-hidden="true"></span>
+            <template #tooltip> {{ t('Discussions') }}</template>
+          </Tooltip>
+        </AdminTableTh>
+        <AdminTableTh
+          :sortable="true"
+          @sort="(direction) => updateSort('views', direction)"
+          :sorted="sorted('views')"
+          scope="col"
+        >
+          <Tooltip>
+            <span class="fr-icon--sm fr-icon-eye-line" aria-hidden="true"></span>
+            <template #tooltip> {{ t('Views') }}</template>
+          </Tooltip>
+        </AdminTableTh>
+        <AdminTableTh scope="col">
+          <Tooltip>
+            <span class="fr-icon--sm fr-icon-download-line" aria-hidden="true"></span>
+            <template #tooltip> {{ t('Downloads') }}</template>
+          </Tooltip>
+        </AdminTableTh>
+        <AdminTableTh
+          :sortable="true"
+          @sort="(direction) => updateSort('reuses', direction)"
+          :sorted="sorted('reuses')"
+          scope="col"
+        >
+          <Tooltip>
+            <span class="fr-icon--sm fr-icon-line-chart-line" aria-hidden="true"></span>
+            <template #tooltip> {{ t('Reuses') }}</template>
+          </Tooltip>
+        </AdminTableTh>
+        <AdminTableTh
+          :sortable="true"
+          @sort="(direction) => updateSort('followers', direction)"
+          :sorted="sorted('followers')"
+          scope="col"
+        >
+          <Tooltip>
+            <span class="fr-icon--sm fr-icon-star-s-line" aria-hidden="true"></span>
+            <template #tooltip> {{ t('Favorites') }}</template>
+          </Tooltip>
+        </AdminTableTh>
+      </tr>
+    </thead>
+    <tbody>
+      <tr v-for="dataset in datasets" :key="dataset.id">
+        <AdminTableTd>
+          <AdminContentWithTooltip>
+            <a class="fr-link" :href="getDatasetLinkToAdmin(dataset)">
+              {{ dataset.title }}
+            </a>
+          </AdminContentWithTooltip>
+        </AdminTableTd>
+        <AdminTableTd>
+          <AdminBadge :type="getStatusType(dataset)">{{ getStatusLabel(dataset) }}</AdminBadge>
+        </AdminTableTd>
+        <AdminTableTd>
+          {{ formatDate(dataset.created_at) }}
+        </AdminTableTd>
+        <AdminTableTd>
+          {{ formatDate(dataset.last_modified) }}
+        </AdminTableTd>
+        <AdminTableTd>
+          {{ getFilesCount(dataset) }}
+        </AdminTableTd>
+        <AdminTableTd>
+          <QualityScoreTooltip :dataset="dataset"/>
+        </AdminTableTd>
+        <AdminTableTd class="monospace">
+          {{ summarize(dataset.metrics.discussions) }}
+        </AdminTableTd>
+        <AdminTableTd class="monospace">
+          {{ summarize(dataset.metrics.views) }}
+        </AdminTableTd>
+        <AdminTableTd class="monospace">
+          {{ summarize(dataset.metrics.downloads ?? 0) }}
```

Comment:
`metrics.downloads` is not defined in types

Reply:
updated to the correct one and a PR is done to add it to the type from datagouv/components too